.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,7 @@
+MARATHON-<<n>> <<Replace this line with your Revision Title>>
+
+Summary:
+
+Test Plan:
+
+JIRA issues:
```

Comment:
It looks like JIRA issue number will be doubled at the beginning and at the bottom.

Reply:
True. I was unsure because we sometimes have multiple issue numbers and this would follow the old pattern a little bit.

Reply:
has anyone ever filled out `Test Plan:` with anything meaningful?

Reply:
@kensipe I use this field usually when I want to highlight a special test case, for example I do:
`sbt 'test-only *PortsMatcherTest'` or I when something should be tested manually. But I also see maybe 60% only `sbt test && sbt integration:test` there.

Reply:
I thought we agreed in the meeting that we would remove `Test Plan:`.